mappings/net/minecraft/server/function/FunctionManager.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_5349 net/minecraft/server/function/FunctionManager
+	FIELD field_25326 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_25327 PATH_PREFIX_LENGHT I
+	FIELD field_25328 PATH_SUFFIX_LENGHT I
+	FIELD field_25329 functions Ljava/util/Map;
+	FIELD field_25330 tags Lnet/minecraft/class_3503;
+	FIELD field_25331 level I
+	FIELD field_25332 commandDispatcher Lcom/mojang/brigadier/CommandDispatcher;
+	METHOD <init> (ILcom/mojang/brigadier/CommandDispatcher;)V
+		ARG 1 level
+		ARG 2 commandDispatcher
+	METHOD method_29447 getFunctions ()Ljava/util/Map;
+	METHOD method_29450 readLines (Lnet/minecraft/class_3300;Lnet/minecraft/class_2960;)Ljava/util/List;
+		ARG 0 resourceManager
+		ARG 1 id
+	METHOD method_29456 get (Lnet/minecraft/class_2960;)Ljava/util/Optional;
+		ARG 1 id
+	METHOD method_29458 getTags ()Lnet/minecraft/class_3503;
+	METHOD method_29459 getOrCreateTag (Lnet/minecraft/class_2960;)Lnet/minecraft/class_3494;
```

Comment:
Imo just `getTag` if it returns an empty tag for an nonexistent one.

Reply:
![](https://s.modm.us/java_2020-06-03_22-41-27.png)

Its very basic, just calls tags.getOrCreate so I think this is fine.